tern/report/report.py

```diff
@@ -152,7 +154,7 @@ def report_out(args, *images):
     report = generate_report(args, *images)
     if not report:
         logger.error("%s not a recognized plugin.", args.report_format)
-    elif args.output_file:
-        write_report(report, args)
+    elif args.output_file or args.report_format == 'html':
```

Comment:
This change looked funny to me for some reason and the reason why is https://github.com/tern-tools/tern/issues/731, which I just opened today. I am going to open a PR to fix issue 731. If it gets merged, you won't need this check here. Stay tuned.